src/main/java/com/autotune/utils/AnalyzerConstants.java

```diff
@@ -134,6 +134,33 @@ public enum ResourceSetting {
         limits
     }
 
+    public enum RecommendationCategory {
+        duration_based,
+        profile_based
+    }
+
+    public enum RecommendationNotificationTypes {
+        INFO ("info", 1),
+        WARN ("warning", 2),
+        ERROR ("error", 3);
```

Comment:
We have a `CRITICAL` that is missing here

Reply:
Sure will be adding that

Reply:
Thanks for the review @dinogun I have added `CRITICAL` type as well